css/types/counters.json

```
@@ -25,22 +25,22 @@
               "version_added": "8"
             },
             "opera": {
-              "version_added": true
+              "version_added": "10"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "10.1"
             },
             "safari": {
-              "version_added": true
+              "version_added": "3.1"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "3.1"
```

And if the other Safari data is true, then this should be 1

```suggestion
              "version_added": "1"
```